fahari/common/migrations/0021_auto_20210911_1724.py

```diff
@@ -19,7 +19,7 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='system',
             name='pattern',
-            field=models.CharField(choices=[('poc', 'Point of Care'), ('rde', 'Retrospective Data Entry'), ('hybrid', 'Hybrid'), ('none', 'None')], default='none', max_length=10),
+            field=models.CharField(choices=[('poc', 'Point of Care'), ('rde', 'Retrospective Data Entry'), ('hybrid', 'Hybrid'), ('none', 'None')], default='none', max_length=100),
```

Comment:
Remove this changes, *never* modify existing migrations, and especially the ones that have been shipped. If you feel changes have to made, modify the models and the rerun `makemigrations`.